--- build/generated/src/main/java/bio/terra/cda/generated/model/QueryResponseData.java.orig	2020-12-17 09:42:40.000000000 -0500
+++ build/generated/src/main/java/bio/terra/cda/generated/model/QueryResponseData.java	2020-12-17 09:42:23.000000000 -0500
@@ -3,6 +3,7 @@
 import java.util.Objects;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonRawValue;
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import java.util.ArrayList;
@@ -21,6 +22,7 @@
   private String querySql;
 
   @JsonProperty("result")
+  @JsonRawValue
   @Valid
   private List<Object> result = null;
 
